### PR TITLE
Set extra tags erlang

### PR DESCRIPTION
**Pull request by jvanbaarsen**
Make sure we set the Hostname and HostMetric tag for all the gauges in the Erlang Probe.

### CODE REVIEW

File: lib/appsignal/probes/erlang_probe.ex [hunk 1]
```diff
@@ -10,31 +10,45 @@ defmodule Appsignal.Probes.ErlangProbe do
...
-      @appsignal.set_gauge("erlang_memory", Kernel.div(value, 1024), %{type: to_string(key)})
+      set_gauge("erlang_memory", Kernel.div(value, 1024), %{type: to_string(key)})
     end)
   end
+
+  defp set_gauge(name, value, tags) do
+    @appsignal.set_gauge(
+      name,
+      value,
+      Map.merge(tags, %{host_metric: "", hostname: hostname()})
```
Comment (tombruijn) [inline lib/appsignal/probes/erlang_probe.ex]: Let's remove the `host_metric: ""`. It's not a host_metric. It's a host specific metric, but not a [host metric](https://docs.appsignal.com/metrics/host.html)

Comment (jvanbaarsen) [reply lib/appsignal/probes/erlang_probe.ex]: Hang on, now I'm confused. This means you're getting back on what you said earlier, correct?

File: lib/appsignal/probes/erlang_probe.ex [hunk 2]
```diff
@@ -10,31 +10,45 @@ defmodule Appsignal.Probes.ErlangProbe do
...
+    )
+  end
+
+  defp hostname do
+    {:ok, hostname} = :inet.gethostname()
+    config = Application.get_env(:appsignal, :config)
+    config[:hostname] || hostname
```
Comment (jeffkreeftmeijer) [inline lib/appsignal/probes/erlang_probe.ex]: We're asking for `:inet.gethostname/0` now, even if we have a configured hostname. Let's check that config value first, which would look something like this (beware, untested):

``` elixir
case Application.fetch_env!(:appsignal, :config)[:hostname] do
  nil -> 
    {:ok, hostname} = :inets.gethostname()
    hostname
  hostname -> hostname
end
```

File: lib/appsignal/probes/erlang_probe.ex (hunk 1 above)
Comment (jvanbaarsen) [reply lib/appsignal/probes/erlang_probe.ex]: Oh wait, after reading back your comment I realise it is about the `void appsignal_set_host_gauge(appsignal_string_t, double);` suggestion.

**Comment by jeffkreeftmeijer** [comment]
Changed the base to master, as this is a hotfix.

**Pull request merged**